### ISSUE TITLE
Account banned

### ISSUE THREAD

**Issue by snowflowercat** [Axcent-ape/Blum#9]
I just create the bot api. then i wait about 1 week and run your code and got banned immediately @@ Why is that ?

**Comment by prono69** [comment]
Telegram account?

**Comment by snowflowercat** [comment]
yes. because the application is created based on the telegram account then my account is banned not only in that game

**Comment by AtamaniukIvan** [comment]
how is a banned account displayed?

**Comment by snowflowercat** [comment]
it does not show anything. It run smoothly until reach resolve peer bot then just got an error in code and my account log in another device logged out. when I log in again, it said my acc is banned.

**Comment by JyuVioleGraceBAM** [comment]
Do you have a print of this screen? Are you sure it was because of the bot and not another reason? Has anyone else had this problem?
